front/lib/api/assistant/rate_limits.ts

```diff
@@ -0,0 +1,36 @@
+import type {
+  LightWorkspaceType,
+  MaxMessagesTimeframeType,
+} from "@dust-tt/types";
+import { expireRateLimiterKey } from "@dust-tt/types";
+
+import type { Authenticator } from "@app/lib/auth";
+
+export const makeMessageRateLimitKeyForWorkspace = (
+  owner: LightWorkspaceType
+) => {
+  return `postUserMessage:${owner.sId}`;
```

Comment:
`nit`: name is a bit misleading no?

Reply:
Could we follow the same pattern as the key below?

Reply:
What do you mean?

Reply:
My bad thought you created the key